executor/ddl.go

```diff
@@ -240,6 +240,7 @@ func (e *DDLExec) executeRenameTable(s *ast.RenameTableStmt) error {
 			return ddl.ErrUnsupportedLocalTempTableDDL.GenWithStackByArgs("RENAME TABLE")
 		}
 		newIdent := ast.Ident{Schema: s.TableToTables[0].NewTable.Schema, Name: s.TableToTables[0].NewTable.Name}
+		isAlterTable := true
```

Comment:
the point is the error message is not the same with MySQL. We'd better change the error message.

BTW, in the previous code, `isAlterTable` is always false. I think it can be removed if you found the commit which introduced this variable.